pkg/audit/verifier.go

```diff
@@ -489,24 +513,42 @@ func (verifier *Verifier) GetShare(ctx context.Context, limit *pb.AddressedOrder
 	}, nil
 }
 
+// RemoveFailedPieces removes lost pieces from a pointer
+func (verifier *Verifier) RemoveFailedPieces(ctx context.Context, path string, pointer *pb.Pointer, failedPieces storj.NodeIDList) (err error) {
```

Comment:
should we add something like
```
if path == "" {
    return nil
}
```
?

Reply:
yeah, I sort of didn't want to, but I've caved on this and added it around other path logic as well.